clients/gax/mix.exs

```diff
@@ -7,7 +7,7 @@ defmodule GoogleApi.Gax.MixProject do
     [
       app: :google_gax,
       version: @version,
-      elixir: "~> 1.6",
+      elixir: "~> 1.9",
```

Comment:
I tried this locally, and since it will always choose poison 5.0.0 I was unable to get the test suite to run with 1.9 or 1.10 (it did work with 1.11).